cmd/terramate/cli/cli.go

```diff
@@ -108,20 +104,27 @@ type cliSpec struct {
 //
 // If a critical error is found an non-nil error is returned.
 func Run(
-	wd string,
 	args []string,
 	inheritEnv bool,
 	stdin io.Reader,
 	stdout io.Writer,
 	stderr io.Writer,
 ) error {
-	c, err := newCLI(wd, args, inheritEnv, stdin, stdout, stderr)
+	c, err := newCLI(args, inheritEnv, stdin, stdout, stderr)
 	if err != nil {
 		return err
 	}
 	return c.run()
 }
 
+type project struct {
+	root    string
+	wd      string
+	isRepo  bool
+	cfg     hcl.Config // root config
```

Comment:
```suggestion
	rootcfg     hcl.Config
```
Would make it more explicitely when referencing it + no need for the docs then.